### PR TITLE
Add an environment variable for bazel-test-integration on Arm

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

**What type of PR is this?**
> /kind bug

**What this PR does / why we need it**:
Add an environment variable for bazel-test-integration on Arm, so we can run it on Arm.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
None
**Does this PR introduce a user-facing change?**:
None
```release-note
None
```

### CODE REVIEW

**Comment by lubinsz** [comment]
@ixdy 
Can you check it?
Thanks.

File: build/root/Makefile [hunk 1]
```diff
@@ -584,6 +585,9 @@ define BAZEL_TEST_INTEGRATION_HELP_INFO
 endef
 bazel-test-integration:
 	@echo "$$BAZEL_TEST_INTEGRATION_HELP_INFO"
+else ifeq ($(UNAME_M),aarch64)
+bazel-test-integration:
+	bazel test --action_env=ETCD_UNSUPPORTED_ARCH=arm64 --config integration //test/integration/...
```
Comment (ixdy) [inline build/root/Makefile]: would it be better to set this environment variable in the test framework itself, i.e. in `test/integration/framework/etcd.go`?

also, is this environment variable documented somewhere?

Comment (lubinsz) [reply build/root/Makefile]: Done.
Please see the document: https://github.com/etcd-io/etcd/blob/a621d807f061e1dd635033a8d6bc261461429e27/Documentation/op-guide/supported-platform.md

**Review by fejta — APPROVED** [review]
/ok-to-test
/lgtm
/approve

**Pull request merged**